### PR TITLE
task/EUI-7322 role category changes

### PR DESCRIPTION
**Pull request by Kartheek-S**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/EUI-7322

### Change description ###
Rework logic for determining user type

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ x ] No
```

### CODE REVIEW

**Review by Mo-Lala84 — APPROVED** [review]
approved

File: api/user/index.ts [hunk 1]
```diff
@@ -35,8 +38,8 @@ export async function getUserDetails(req, res: Response, next: NextFunction): Pr
   }
 }
 
-export async function refreshRoleAssignmentForUser(userInfo: UserInfo, req: any): Promise<any []> {
-  let locationInfo = [];
+export async function refreshRoleAssignmentForUser(userInfo: UserInfo, req: any): Promise<any[]> {
+  let roleAssignmentsInfo = [];
```
Comment (udaydenduluri33) [inline api/user/index.ts:42]: Please rename it to userRoleAssignments

**Review by udaydenduluri33 — APPROVED** [review]
LGTM 
It would be great if you could do 2 minor changes

**Review by udaydenduluri33 — APPROVED** [review]

**Review by DavidJayakumar — APPROVED** [review]

**Review by andywilkinshmcts — DISMISSED** [review]

**Review by DavidJayakumar — DISMISSED** [review]
LGTM

**Review by tarikdogan-hmcts — APPROVED** [review]

**Comment by Kartheek-S** [comment]
Closing this PR because changes have already merged into the master branch

**Pull request closed without merging**